src/App.vue

```diff
@@ -26,11 +31,25 @@ export default {
 
   methods: {
     addTodo (newTodo) {
-      this.todos.push(newTodo)
+      let todo = {
+        title: newTodo,
+        completed: false,
+        editing: false
```

Comment:
you don't need `completed: false` and `editing: false` for now

Reply:
Removed from todo.